plugins/kubernetes-cluster/src/components/KubernetesClusterContent/KubernetesClusterContent.tsx

```diff
@@ -57,7 +59,15 @@ const ContentGrid = () => {
  * @public
  */
 export const KubernetesClusterContent = () => {
-  return (
+  const hasKubernetesClustersPermission = useKubernetesClustersPermission();
+
+  return !hasKubernetesClustersPermission ? (
```

Comment:
mmm I think this will cause a glitch where `WarningPanel` will be visible while the permission is loading